code/modules/admin/admin_verbs.dm

```diff
@@ -794,6 +797,18 @@ var/list/admin_verbs_cciaa = list(
 			verbs |= /client/proc/readmin_self
 	feedback_add_details("admin_verb","DAS") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/client/proc/toggle_aooc()
+	set name = "Toggle AOOC"
+	set category = "Admin"
+	if(holder)
+		if (tao() == 0)
```

Comment:
```suggestion
		if (tao() == FALSE)
```

Please for the love of all that is holy, use descriptive proc/variable names. To someone that hasn't read everything, "TAO" and "togantag" mean nothing.

Reply:
Changed.